### ISSUE TITLE
Nette 2.3 support

### ISSUE THREAD

**Issue by meridius** [jsmitka/Nette-XDebug-Helper#2]
Hi,

would it be possible to add support for Nette 2.3? 
When I init the extension by adding following to the config of Nette 2.3.3:

```
tracy:
    bar: [NetteExtras\XDebugHelper()]
```

the whole Tracy panel disappears and I have to init it manually by calling the content of `window.addEventListener('load')` function. After that the extension is working, so this is the only drawback.
I also tried to tinker with the code but with no luck.

**Comment by kyborek** [comment]
Hello, i have been able to easily integrate this into 2.3.5 nette version with slightly different syntax in my config.neon file:

```
tracy:
    bar:
        - NetteExtras\XDebugHelper
```

I have even tried your syntax and it works for me too, are you sure there is no other issue? Can you try adding the extension to plain sandbox application?

**Comment by meridius** [comment]
You are right, both syntaxes work on a clean sandbox. I've narrowed down the problem to my meridius/yadup extension, it affects it somehow. I've looked at the code of nette-xdebug-helper and there shouldn't be anything yadup could break. I guess I've to keep looking. :/

**Comment by JanGalek** [comment]
I sent PR https://github.com/jsmitka/Nette-XDebug-Helper/pull/4

you can wait for merge, or use in composer:

> 
	"repositories": [
		{
			"type": "git",
			"url": "https://github.com/JanGalek/Nette-XDebug-Helper"
		}
	],
